### DISCUSSION TITLE
Random anime/manga button

### DISCUSSION THREAD

**Discussion by xLexih** [aniyomiorg/aniyomi#785] [Ideas]
It would be cool if there was a way to get some random animes directly from the app maybe a button that would show something similar to https://www.randomanime.org/ I'm not sure if this is hard or possible to implement but it would be cool :)

**Comment by Noorislam-XD** [comment]
It would be great if it's added.

**Comment by Samfun75** [comment]
Update to the latest beta and you will have this,

![Screenshot_20221206-092817_Aniyomi](https://user-images.githubusercontent.com/38332931/205837918-f85ecae7-fe1c-403d-9d65-7ebbb0b4e88c.jpg)

**Reply by Kakarot2005** [reply]
How can I update to latest beta?

**Reply by Samfun75** [reply]
> How can I update to latest beta?

https://github.com/jmir1/aniyomi-mpv-beta/releases
